lib/main.coffee

```diff
@@ -95,8 +95,23 @@ module.exports =
     getLeftTiles: @statusBar.getLeftTiles.bind(@statusBar)
     getRightTiles: @statusBar.getRightTiles.bind(@statusBar)
 
-  # Depreciated method associated with previous Services API
-  # versioning that matched package version.
+  # Depreciated
+  #
+  # Wrap deprecation calls on the methods returned rather than
+  # Services API method which would be registered and trigger
+  # a deprecation call
   legacyProvideStatusBar: ->
-     Grim.deprecate("Use versions ^1.0.0 of status-bar Service API.")
-     @provideStatusBar()
+    sb = @provideStatusBar()
+
+    addLeftTile: (args...) ->
+      Grim.deprecate("Use versions ^1.0.0 of status-bar Service API.")
```

Comment:
Just rereading this message, should versions be singular instead? and maybe add a `the` before `status-bar?